plugins/scaffolder-react/src/api/ref.ts

```diff
@@ -16,13 +16,8 @@
 
 import { createApiRef } from '@backstage/core-plugin-api';
 import { ScaffolderApi } from './types';
-import { getOrCreateGlobalSingleton } from '@backstage/version-bridge';
```

Comment:
Do we still need this dep?

Reply:
Yep it's used for a React context singleton